StardewArchipelago/Constants/ModNames.cs

```diff
@@ -27,8 +27,8 @@ public class ModNames
         public const string ALECTO = "Alecto the Witch";
         public const string DISTANT_LANDS = "Distant Lands - Witch Swamp Overhaul";
 
-        public const string AP_SVE = "StardewArchipelago - Stardew Valley Expanded";
-        public const string AP_ALECTO = "StardewArchipelago - Alecto the Witch";
-        public const string AP_JASPER = "StardewArchipelago - Professor Jasper Thomas";
+        public const string AP_SVE = "StardewAP - SVE Patch";
+        public const string AP_ALECTO = "StardewAP - Alecto the Witch Patch";

```

Comment:
why not "Alecto Patch" and "Jasper Patch", to simplify the same as much as possible and reduce similarity with the original while maintaining clarity?

Reply:
Because then its too generic with any other patch.  It should say what the patch is for.

Reply:
Possibly I didn't express this right.
I meant replace "Alecto the Witch Patch" with "Alecto Patch". So the full mod name would go from "StardewAP - Alecto the Witch Patch" to "StardewAP - Alecto Patch", which I think would be just as specific as the current name, but less wordy.

I did not mean to get rid of the "StardewAP - " part

Reply:
A bit tedious to do again, but alright.